decentraland/unity-renderer
# fix: load empty lands on the builder panel

## What does this PR change?

This PR fixes the problem where if you try to load the scene entity from an empty parcel where a deploy never occurs, the builder project panel never load anything.

Fixes #897 

## How to test the changes?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/lots-of-lands
2. Enter with an account that has an empty land where a deploy has never occurs
3. Check that the builder panel loads correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
